javascript/commons/SwitchButtons.js

```diff
@@ -144,6 +158,10 @@ liquipedia.switchButtons = {
 
 	attachEventListener: function ( switchGroup, activeClassName ) {
 		switchGroup.nodes.forEach( ( node ) => {
+			if ( switchGroup.name === 'matchFiler' && switchGroup.value === 'completed' ) {
```

Comment:
is this supposed to be `matchFiler` or is that a typo from `matchFilter`?

Reply:
Haha yea the typo already existed in the `Container.lua` file. I changed it to `matchFilter` now, don't think that breaks anything 😄